wicket-core/src/main/java/org/apache/wicket/markup/head/CssHeaderItem.java

```diff
@@ -169,6 +171,39 @@ public static CssReferenceHeaderItem forReference(ResourceReference reference,
 	}
 
 	/**
+	 * Creates a {@link CssReferenceHeaderItem} for the given reference.
+	 *
+	 * <strong>Warning</strong>: the conditional comments don't work when injected dynamically with
```

Comment:
I just copied the constructor and added the rel parameter. Thats the reason why it is applied there, too. I would just keep it to be backward compatible - the arguments are all Strings, so I have to add a new one behind. 😞